sympy/physics/vector/frame.py

```diff
@@ -387,8 +388,8 @@ def ang_acc_in(self, otherframe):
         Parameters
         ==========
 
-        otherframe : ReferenceFrame
-            The ReferenceFrame which the angular acceleration is returned in.
+        otherframe : ReferenceFrame or Body
+            The ReferenceFrame or Body which the angular acceleration is returned in.
 
```

Comment:
These changes should be done in public function where it is called, this function should receive frame only, unless there is an issue with it.

Reply:
> These changes should be done in public function where it is called, this function should receive frame only, unless there is an issue with it.

I am not able to understand this, the function `ang_acc_in()` in which this change is made is already public. In which functions do I have to make changes?